modules/RTC/JitsiRemoteTrack.js

```diff
@@ -311,6 +324,6 @@ export default class JitsiRemoteTrack extends JitsiTrack {
      */
     toString() {
         return `RemoteTrack[userID: ${this.getParticipantId()}, type: ${this.getType()}, ssrc: ${
-            this.getSSRC()}, p2p: ${this.isP2P}, status: ${this._getStatus()}]`;
+            this.getSSRC()}, p2p: ${this.isP2P}, sourceName: ${this.sourceName}, status: ${this._getStatus()}]`;
```

Comment:
what do you think about changing it to ` n: ${this.sourceName}`? I'm thinking the shorter the better, given that it's getting crowded in here already

Reply:
I saw similar single letter naming in the compact json structure.  Was concerned single letter naming conventions (granted this is logging here and not variable naming) could be confusing when debugging (greps for 'sourceName' as such).  I can make that adjustment if you're ok with it.

Reply:
ok lets keep it this way